frontend/src/pages/sell/character-sell.tsx

```diff
@@ -13,13 +15,22 @@ export const CharacterSell = () => {
   const sellCharacter = useSellCharacter(Number(idString));
   const [character] = useMyCharacter(Number(idString));
   const [characterCopy] = useState(character);
+  const userDispatch = useUserStateDispatch();
 
   const [isPlacedInShop, setIsPlacedInShop] = useState(false);
   const [data, setData] = useState<SellData>({ price: 0 });
 
+  const handleResult: MakeOfferCallback = {
+    settled: () => {
+      // Currently calling this logic on settled due to an issue with the status returned by the sell method,
```

Comment:
We can actually fix this todo now that we have the `seated` callback

Reply:
Ah yes good point. Fixed!